Cloud Enterprise/Getting Started Examples/aws/terraform/ansible-install.sh

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env bash
+
+# Short form: set -u
+set -o nounset
+# Short form: set -e
+set -o errexit
+
+# Print a helpful message if a pipeline with non-zero exit code causes the
+# script to exit as described above.
+trap 'echo "Aborting due to errexit on line $LINENO. Exit code: $?" >&2' ERR
+
+# Allow the above trap be inherited by all functions in the script.
+#
+# Short form: set -E
+set -o errtrace
+
+# Return value of a pipeline is the value of the last (rightmost) command to
+# exit with a non-zero status, or zero if all commands in the pipeline exit
+# successfully.
+set -o pipefail
+
+# Set $IFS to only newline and tab.
+#
+# http://www.dwheeler.com/essays/filenames-in-shell.html
+IFS=$'\n\t'
+
+###############################################################################
+# Program Functions
+###############################################################################
+_write_ansible_playbook() {
+cat << PLAYBOOK > ./ece.yml
+---
+- hosts: primary
+  gather_facts: true
+  roles:
+    - ansible-elastic-cloud-enterprise
```

Comment:
Will this just work ? Wouldn't the user have to install the role ?

Reply:
Ah. Thats correct. 
Missed this one, i'll include role-installing in the ansible-installer.sh